### PR TITLE
hello example with clean arch

### PR DESCRIPTION
**Pull request by huhuhu0420**

### CODE REVIEW

**Comment by nonorze-VOXX** [comment]
get "gopher" success on my computer

File: backend/go-server/hello/usecase/hello_usecase.go [hunk 1]
```diff
@@ -0,0 +1,27 @@
+package usecase
+
+import (
+	"context"
+	"main/domain"
+
+	"github.com/sirupsen/logrus"
+)
+
+type helloUsecase struct {
```
Comment (nonorze-VOXX) [inline backend/go-server/hello/usecase/hello_usecase.go:10]: why your empty is 8

Comment (huhuhu0420) [reply backend/go-server/hello/usecase/hello_usecase.go:10]: ha?

File: backend/go-server/go.mod [hunk 2]
```diff
@@ -0,0 +1,33 @@
+module main
```
Comment (NOOBDY) [inline backend/go-server/go.mod:1]: usually golang modules should be named with their version control repo name (i.e. `module github.com/PUArallelepiped/PUN-street-Universal-Access`)

Comment (huhuhu0420) [reply backend/go-server/go.mod:1]: k, you're right

**Pull request closed without merging**